model/src/main/kotlin/ProjectAnalyzerResult.kt

```diff
@@ -19,20 +19,15 @@
 
 package com.here.ort.model
 
-import com.fasterxml.jackson.annotation.JsonAlias
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 
 import java.util.SortedSet
 
 /**
  * A class that bundles all information generated during an analysis.
  */
+@JsonIgnoreProperties("allowDynamicVersions", "allow_dynamic_versions", "config")
```

Comment:
Is this still needed as the `AnalyzerResult` is not a collection of `ProjectAnalyzerResult`s, and `ProjectAnalyzerResult` is not deserialized on its own anymore?